condor_jobs/submit.sh

```diff
@@ -1,31 +1,37 @@
 ###############################
 # Submit shape jobs to condor #
 ###############################
-# create error log out folder if not existens
-mkdir -p error log out
-
+# create folder if it doesn't exit
+dir=output/log/condorShapes/
+[[ -d $dir ]] || mkdir -p $dir
 # jobs will be submitted directly from this repository, so the logfiles created by the job will be updated.
-ERA=$1
-CHANNELS=${@:2}
-PWD=`pwd`
-# write arguments.txt
-WORKDIR=`dirname $PWD`
-
-for CHANNEL in $CHANNELS 
-do  
-    echo "$WORKDIR $ERA $CHANNEL"
-done > arguments.txt
-
```

Comment:
Before we could submit all 4 channels in one call: ./condor_jobs/submit.sh 2017 mt et tt em, this was very convenient. I think like this this will not be possible anymore?

Reply:
with  `./condor_jobs/submit.sh 2017 mt,et,tt,em` , yes it is still possible

Reply:
okay nice